README: Moving cppcheclipse development to new location (#116)
kwin/cppcheclipse: README.md

![Java CI](https://github.com/kwin/cppcheclipse/workflows/Java%20CI/badge.svg)

**IMPORTANT:**

> This is the original cppcheclipse repository. It will not be actively developed anymore.

Reviewer: ```suggestion
This is the original cppcheclipse repository. It is not actively maintained in this repository anymore.
```

> Latest cppcheclipse source code is available [here](https://github.com/cppchecksolutions/cppcheclipse). Please report issues and provide pull requests in that repository.

Reviewer: ```suggestion
You now find the cppcheclipse source code [here](https://github.com/cppchecksolutions/cppcheclipse). Please report issues and provide pull requests in that repository instead.
```
